lib/core/form/components/widgets/invalid-error-state-matcher.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FormControl  } from '@angular/forms';
+import { ErrorStateMatcher } from '@angular/material/core';
+
+export class InvalidErrorStateMatcher implements ErrorStateMatcher {
+    isErrorState(control: FormControl | null): boolean {
```

Comment:
can we add some unit tests? If I modify this function I should see a red unit test